scripts/managedb/csub_merge_condvalues.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+# -*- coding: latin-1 -*-
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+
+# This script is inspired by old attrsvector2attrsobject.py
+
+__author__ = 'fermin'
+
+from pymongo import MongoClient
+import json
+import sys
+from time import sleep
+
+CONDITIONS = "conditions"
+
+def update_ok(doc, n_conditions):
+
```

Comment:
Fixed in 0dc11b3 (not only for csub_merge_condvalues.py, but also for in mdsvector2mdsobject.py).